backend/redis/utils.go

```diff
@@ -532,3 +532,10 @@ func (r *Client) FlushDB(ctx context.Context) error {
 	}
 	return nil
 }
+
+func (r *Client) Del(ctx context.Context, key string) error {
```

Comment:
will this method be used by a test for clearing a cache key?

Reply:
Correct. The tests are down below.